bin/inx-chronicle/src/main.rs

```diff
@@ -12,6 +12,7 @@ mod collector;
 mod config;
 #[cfg(feature = "inx")]
 mod inx;
+mod metrics;
```

Comment:
I guess the plan is to also feature-gate this?

Reply:
Should be